Tests/UnitTests/Misc/CustomerInfo+TestExtensions.swift

```diff
@@ -38,7 +38,8 @@ extension CustomerInfo {
             "original_app_user_id": "app_user_id",
             "subscriptions": [:] as [String: Any],
             "other_purchases": [:] as [String: Any],
-            "original_application_version": "1.0"
+            "original_application_version": "1.0",
+            "original_purchase_date": "2019-07-17T00:05:54Z"
```

Comment:
Forgot to add a comment about this:

The only test where not setting the date changes the result of a test is [testSyncPurchasesDoesntPostReceiptAndReturnsCustomerInfoIfNoTransaction](https://github.com/RevenueCat/purchases-ios/pull/3699/files#diff-de8f414dec0c070ee1166c6954bb01a58ae0bce1d1c7f14b950598769847e513R586) because if there is no `original_purchase_date`, we are [posting the receipt](https://github.com/RevenueCat/purchases-ios/blob/main/Sources/Purchasing/Purchases/PurchasesOrchestrator.swift#L1050) even if there are no transactions.

I added a more explicit test for that behavior in `testSyncPurchasesSK1PostsReceiptIfNoTransactionsAndEmptyOriginalPurchaseDate`.

I think this was not explicitly tested before 😅